src/ape_geth/providers.py

```diff
@@ -295,7 +295,17 @@ def get_transaction_trace(self, txn_hash: str) -> Iterator[TraceFrame]:
 
     def get_call_tree(self, txn_hash: str, **root_node_kwargs) -> CallTreeNode:
         def _get_call_tree_from_parity():
-            raw_trace_list = self._make_request("trace_transaction", [txn_hash]).get("results", [])
+            response = self._make_request("trace_transaction", [txn_hash])
+            if "error" in response:
+                raise ProviderError(
+                    response["error"].get("message", f"Failed to get trace for '{txn_hash}'.")
+                )
+
+            raw_trace_list = response.get("result", [])
+
+            if not raw_trace_list:
```

Comment:
have to do this because I see weird things where some transactions have 0 trace and `get_calltree_from_parity_trace` expects at least one item in the result.